components/gcp-click-to-deploy/app-config.yaml

```diff
@@ -18,6 +18,8 @@ defaultApp:
     prototype: cert-manager
   - name: iap-ingress
     prototype: iap-ingress
+  - name: proxy-agent
+    prototype: proxy-agent
```

Comment:
proxy-agent can be an optional components based on user's choice of how to access UI.
We should make the change on js side.

Reply:
is this file for local test only? if not adding it, will local test be able to deploy proxy agent component?

Reply:
Yeah, this one is for local test only.
We can edit config in javascript based on a param?